### ISSUE TITLE
Emoji Text representations

### ISSUE THREAD

**Issue by ifsimicoded** [cityofaustin/techstack#327]
For the Feedback Form there are emojis which need more usable text representations for screen readers. These should be culturally conscious choices. Our current text is based on unicode names for emojis but isn't user friendly. The text is as follows:
😞 Disappointed
🙁 Sad
😐 Neutral
🙂 Slightly Smiling
😀 Grinning 

1. Get text representations of emojis
2. Translate text representations
3. Implement text and translations on Janis

**Comment by cthibaultatx** [comment]
@LauraTrujillo hey! I think you mocked the alternative text up for the emojis somewhere. Can you copy+paste here? I'm going to document in the pattern library...or someone can at some point.

**Comment by cthibaultatx** [comment]
Here's what I mocked up in the [sketch design library file](https://drive.google.com/file/d/1v95Zwm7aaXpD45jUCE70vuCtMv9JMedM/view?usp=sharing). Please update as needed.

![screen shot 2018-06-08 at 8 53 25 am](https://user-images.githubusercontent.com/20911665/41161725-6de8cbc6-6af9-11e8-9cb4-e957ab27bc0e.png)

**Comment by amenity** [comment]
Cleaning out ancient, probably-no-longer-applicable, issues. This is captured in #682 .
